### DISCUSSION TITLE
i need help my github.io is not working properly....

### DISCUSSION THREAD

**Discussion by HarshitJain1924** [community/community#116805] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

i have to submit a project on coursera for that i need github.io but i can't access my website on it. 
i tried it for my original website too still i am getting my website as dangerous .
Please Help!

![Screenshot (103)](https://github.com/community/community/assets/121855889/80764ec9-6841-46e1-9467-8e87396af23e)

**Comment by brenoSCosta** [comment]
This issue may have arisen due to the creation of clone repositories for Netflix and Amazon.

As a precautionary measure, I strongly advise against launching clone projects, such as replicas of Netflix and Amazon, within your instance. These replicas pose a risk of being flagged as unsafe or potential phishing sites by Google Safe Browsing, which could ultimately jeopardize user safety.

To resolve this issue:

1. Confirm that you have removed the problematic repository from GitHub.
2. Access Google Search Console and navigate to the Security Issues section.
3. Click on the relevant link or tab labeled "Security Issues."
4. Request a review from Google to verify the takedown of the offending website.
5. Fill out the Incorrect Phishing Warning Google Form to report any discrepancies.
6. Please note that the review process may take several days or longer.

**Reply by brenoSCosta** [reply]
If you need more help, you can talk to me, I went through the same process. I hope it helped you!

**Reply by HarshitJain1924** [reply]
thank you for reply! Issue has been resolved. It was a part of html, css and javascript and i need to clone that particular resturant website. 
i did fill request in google search console.
